js/scripts.js

```diff
@@ -2,16 +2,43 @@ function Phone(brand, price, color) {
 	this.brand = brand;
 	this.price = price;
 	this.color = color;
-}
+		this.getWarrantyCost = function() {
+		return this.price * 0.1;
+		console.log(this.getWarrantyCost);  //jak dodatkowo wyświetlić w konsoli wynik getWarrantyCost?
+		}; //sposób 1
+};	
+
+//Phone.prototype.getWarrantyCost = function() {
+//	return warrantyCost = this.price * 0.1;
+//} //sposób 2
```

Comment:
to będzie ok tylko niepotrzebna ta zmienna `warrantyCost` - po prostu zwracaj `this.price + 0.1`